detox/test/android/app/src/main/java/com/example/DoubleTapsTextViewManager.java

```diff
@@ -0,0 +1,57 @@
+package com.example;
+
+import android.graphics.Color;
+import android.view.GestureDetector;
+import android.view.Gravity;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewGroup.LayoutParams;
+import android.widget.FrameLayout;
+import android.widget.TextView;
+
+import com.example.utils.DoubleTapListenerStub;
+import com.facebook.react.uimanager.SimpleViewManager;
+import com.facebook.react.uimanager.ThemedReactContext;
+
+public class DoubleTapsTextViewManager extends SimpleViewManager<ViewGroup> {
```

Comment:
Can you extend `ReactTextViewManager`? Last time I checked, extending base view managers wasn't possible, perhaps that changed? Then you wouldn't have to mess around layout params and styles in this class.

Reply:
I tried, tbh, and it's too much of a headache. I don't need the manager + shadow-node in this case, since I don't want things such as having the layout and content controlled by the JSX -- it's actually somethings I need to avoid here, and keep content changes "instant".